Lib/turtle.py

```diff
@@ -1492,6 +1493,39 @@ def screensize(self, canvwidth=None, canvheight=None, bg=None):
         """
         return self._resize(canvwidth, canvheight, bg)
 
+    def save(self, filename, overwrite=False):
```

Comment:
I find it can be helpful to make boolean args keyword-only, so you have to call it like `save("example.ps", overwrite=True)`, and not `save("example.ps", True)` -- it's not very clear what that bare `True` means.

But perhaps you have a better idea of which is better for beginners?

```suggestion
    def save(self, filename, *, overwrite=False):
```

Reply:
Hmmm, that's a tricky question! From an API-design standpoint, I favour making it keyword-only. However, functions can be notoriously tricky for beginner coders to grasp, so it's likely that a learner will encounter this save function before learning about keyword-only arguments, which might be confusing. 

On the other hand, though, I've been able to give examples using print with the keyword-only `end` argument to beginners without too much confusion by explaining that this is a more advanced concept they don't need to fully understand yet. So it might be fine and can be a way to preview the concept of keyword-only arguments.

Reply:
Also, it's easier to make the API more flexible in the future than less flexible, so maybe it makes sense to have it keyword-only, and if we get feedback that it's confusing change it to `overwrite` as positional.